<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Registration Page Test"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="4.456">
  <properties />
            <testcase name="verifyThatFirstNameLastNameEmailPasswordAndConfirmPasswordFieldsAreMandatory" time="1.117" classname="com.nopcommerce.demo.testsuite.RegisterPageTest">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	Password field error message not displayed expected [null] but found [Password is required.]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	Password field error message not displayed expected [null] but found [Password is required.]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
  com.nopcommerce.demo.testsuite.RegisterPageTest.verifyThatFirstNameLastNameEmailPasswordAndConfirmPasswordFieldsAreMandatory(RegisterPageTest.java:53)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:568)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
  org.testng.TestRunner.privateRun(TestRunner.java:739)
  org.testng.TestRunner.run(TestRunner.java:614)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
  org.testng.SuiteRunner.run(SuiteRunner.java:312)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
  org.testng.TestNG.runSuites(TestNG.java:1112)
  org.testng.TestNG.run(TestNG.java:1079)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="verifyThatUserShouldCreateAccountSuccessfully" time="2.365" classname="com.nopcommerce.demo.testsuite.RegisterPageTest" />
                <testcase name="verifyUserShouldNavigateToRegisterPageSuccessfully" time="0.974" classname="com.nopcommerce.demo.testsuite.RegisterPageTest" />
      
</testsuite>
